geopy/util.py

```diff
@@ -80,3 +80,10 @@ def decode_page(page):
         else: # requests?
             encoding = page.headers.get("charset") or "utf-8"
             return str(page.content, encoding=encoding)
+
+
+def get_version():
+    import geopy
+    return str(geopy.__version__)
```

Comment:
It's cool, you just need to put it in the init module to have it, e.g., `import geopy.version.GEOPY_VERSION as __version__`, yeah?